docs/root/configuration/http/http_filters/wasm_filter.rst

```diff
@@ -0,0 +1,37 @@
+.. _config_http_filters_wasm:
+
+Wasm
+====
+
+* :ref:`v3 API reference <envoy_v3_api_msg_extensions.filters.http.wasm.v3.Wasm>`
+* This filter should be configured with the name *envoy.filters.http.wasm*.
```

Comment:
Why? We now just infer the extension from the type URL, so you can call the filter whatever you want, which is what we need when there are multiple Wasm filters instantiated.

Reply:
I copied that from docs/root/configuration/http/http_filters/tap_filter.rst

What is the new standard?  Do we want to do a pass and update all the docs?

Reply:
Updated PTAL